### PR TITLE
add chainflip boost

### PR DESCRIPTION
**Pull request by anthon-dev**
- Added the boost feature for the chainflip plugin.
- Tested on playground

### CODE REVIEW

File: packages/plugins/chainflip/src/plugin.ts [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
   sellAsset,
   recipient,
   brokerEndpoint,
+  maxBoostFeeBps,
 }: {
   buyAsset: AssetValue;
   sellAsset: AssetValue;
   recipient: string;
   brokerEndpoint: string;
+  maxBoostFeeBps: number;
```
Comment (ice-chillios) [inline packages/plugins/chainflip/src/plugin.ts:26]: It should be probably optional and defaulted to `0`

File: packages/plugins/chainflip/src/plugin.ts [hunk 2]
```diff
@@ -76,6 +81,7 @@ function plugin({
       buyAsset,
       recipient,
       sellAsset,
+      maxBoostFeeBps: maxBoostFeeBps ?? 0,
```
Comment (ice-chillios) [inline packages/plugins/chainflip/src/plugin.ts]: This condition never "fires" as `maxBoostFeeBps` is required param.

**Review by ice-chillios — CHANGES_REQUESTED** [review]

File: packages/plugins/chainflip/src/plugin.ts (hunk 1 above)
Comment (anthon-dev) [reply packages/plugins/chainflip/src/plugin.ts:26]: done

File: packages/plugins/chainflip/src/plugin.ts (hunk 2 above)
Comment (anthon-dev) [reply packages/plugins/chainflip/src/plugin.ts]: done

**Review by ice-chillios — APPROVED** [review]

**Pull request merged**